docs/sphinx/developer/docker.rst

```diff
@@ -152,7 +154,9 @@ Test & Develop
 
 The Dockerfile is defined to start all Phoniebox related services.
 
-Open `http://localhost:3001 <http://localhost:3001>`_ in your browser to see the web application.
+Windows: Open `host.docker.internal:3000 <http://host.docker.internal:3000>`_ in your browser to see the web application.
```

Comment:
Shouldn't that be also just `http://localhost:3000`?

Reply:
localhost is changed to 0.0.0.0 by jukebox. I noticed that on windows I only have a route that routes traffic from 0.0.0.0 to my physical network adapter, so the jukebox could not connect. On another windows with docker I noticed that a network route from 0.0.0.0 to the docker network adapter exists, so I'm not sure why it is missing on my installation.

Anyhow I figured that using host.docker.internal on windows might work on more installations (including mine), but if you want I'll undo that change. For mac, my impression was that host.docker.internal also exists and should work, but I'm not sure and can't test.

Reply:
`host.docker.internal` does not work for me on Mac. Maybe we can add a comment mentioning both, so it works for everyone depending on their set up.

Reply:
Doc changed for Mac to `localhost:3000`